source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.md

```diff
@@ -81,31 +81,51 @@ parse error and MUST NOT do DNS resolution or contact hosts.
 It is an error to specify more than one host name in a connection string with the `mongodb+srv` protocol, and the driver
 MUST raise a parse error and MUST NOT do DNS resolution or contact hosts.
 
-A driver MUST verify that in addition to the `{hostname}`, the `{domainname}` consists of at least two parts: the domain
-name, and a TLD. Drivers MUST raise an error and MUST NOT contact the DNS server to obtain SRV (or TXT records) if the
-full URI does not consist of at least three parts.
+Node:
 
 If `mongodb+srv` is used, a driver MUST implicitly also enable TLS. Clients can turn this off by passing `tls=false` in
 either the Connection String, or options passed in as parameters in code to the MongoClient constructor (or equivalent
 API for each driver), but not through a TXT record (discussed in a later section).
 
+#### Terminology
+
+For the purposes of this document, `{hostname}` will be divided using the following terminology.
+
+If an SRV `{hostname}` has:
+
+- 3 or more `.` separated parts:
+  - the left-most part is the `{subdomain}`
+  - the remaining portion is the `{domain}`
```

Comment:
We say "domain" here but use "domainname" in other parts of the document, we should pick one and make it consistent everywhere.

Reply:
Let's use `domainname`! Changed the other files to reflect this.